### PR TITLE
Api-Buildout: Update Home Page and Create Featured Stories

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This PR contains changes to the Home Page to render the Featured Stories.
### FeaturedStoryPreview.js:
- renders the story title/quote in 2 parts (regular and italics)
- renders the story preview blurb
- renders the story preview image
- renders the author's signature
- has a link to the story

### index.js
- fetches the featured stories
- renders the featured stories using the FeaturedStoryPreview component

### api.js
- getAuthorSignatureFromIds returns a dictionary containing the Author's Id and the Author's signature
- getFeaturedStories returns an array of featured stories containing the necessary data from Prismic

### CODE REVIEW

**Review by jzhzhang66 — APPROVED** [review]
This looks great!

File: components/FeaturedStoryPreview.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    preview_image: previewImage,
+    signature,
+  } = featuredStoryData;
+  return (
+    <>
+      <H3>{getString(previewTitleRegular)}</H3>
+      <H3>{getString(previewTitleItalic)}</H3>
+      <P>{getString(previewText)}</P>
+      <img
+        style={{ width: '500px' }}
```
Comment (josephmannis) [inline components/FeaturedStoryPreview.js]: Can you pull out this style please?

**Review by josephmannis — CHANGES_REQUESTED** [review]
Just one tiny thing then good to go

File: components/FeaturedStoryPreview.js (hunk 1 above)
Comment (isabellberzsenyi) [reply components/FeaturedStoryPreview.js]: Decided to remove the styling since that will all be done in a separate PR, good catch thank you

**Pull request merged**